hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetInfoHandler.java

```diff
@@ -149,7 +149,8 @@ private Optional<TokenInfo> infoForToken(
             info.symbol(token.symbol());
             info.name(token.name());
             info.memo(token.memo());
-            info.treasury(AccountID.newBuilder().accountNum(token.treasuryAccountNumber()));
+            info.treasury(
+                    AccountID.newBuilder().accountNum(token.treasuryAccountId().accountNum()));
```

Comment:
Could this be `info.treasury(token.treasuryAccountId());` ?

Reply:
Updated, thanks.